content/en/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md

```diff
@@ -13,7 +13,7 @@ Kubernetes' support for direct integration with Docker Engine is deprecated, and
 ## Telemetry and security agents
 
 Within a Kubernetes cluster there are a few different ways to run telemetry or security agents.
-Some agents have a direct  dependency on Docker Engine when they as DaemonSets or
+Some agents have a direct dependency on Docker Engine when they as DaemonSets or
```

Comment:
```suggestion
Some agents directly depend on Docker Engine when they run as DaemonSets or
```

Reply:
There's some technical ambiguity in what this phrase is saying. Please clarify which of the following meanings you intend to convey:
* **Some** (not all) agents that run as DaemonSets or directly on nodes depend on Docker Engine.
* Agents that run as DaemonSets or directly on nodes depend on Docker Engine.